internal/riverinternaltest/riverdrivertest/riverdrivertest.go

```diff
@@ -192,6 +192,26 @@ func Exercise[TTx any](ctx context.Context, t *testing.T,
 		exists, err = exec.ColumnExists(ctx, "river_job", "does_not_exist")
 		require.NoError(t, err)
 		require.False(t, exists)
+
+		exists, err = exec.ColumnExists(ctx, "does_not_exist", "id")
+		require.NoError(t, err)
+		require.False(t, exists)
+
+		// Will be rolled back by the test transaction.
+		_, err = exec.Exec(ctx, "CREATE SCHEMA another_schema_123")
+		require.NoError(t, err)
+
+		_, err = exec.Exec(ctx, "SET search_path = another_schema_123")
```

Comment:
Does this pollute the search path for this connection for future tests? Or does the conn pool get thrown away entirely after each test before the test DB goes back into the DB pool?

Reply:
It gets rolled back with the transaction:

``` sql
river_dev=# show search_path;
                 search_path
---------------------------------------------
 "$user", crunchy_cloud, infra, logs, public
(1 row)


river_dev=# begin;
BEGIN

river_dev=*# set search_path = 'hello';
SET

river_dev=*# show search_path;
 search_path
-------------
 hello
(1 row)

river_dev=*# rollback;
ROLLBACK

river_dev=# show search_path;
                 search_path
---------------------------------------------
 "$user", crunchy_cloud, infra, logs, public
(1 row)
```